02stl/cpp/sortArrays.cc

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
```

Comment:
I don't get this example. It looks like you are trying to teach them C in addition to C++. Is it necessary?